src/components/tooltipUtils.js

```diff
@@ -86,6 +86,10 @@ export function createVegaTooltipAdapter(svelteComponent, initialExtraProps = {}
       return;
     }
 
+    // If the datum value is null, no tooltip is needed.
```

Comment:
should this condition be close to the hide condition on top such that the toolitp is hidden properly?

Reply:
I didn't need to hide it (by calling hide()?) in order to avoid showing it.  I do need to return before the following update() call, or else it would show an empty tooltip.

But I can move this up and combine it with the earlier condition.  Maybe the hide() call is needed if there is already an open tooltip?